openapi.yaml

```diff
@@ -704,7 +704,7 @@ components:
     FiscalCode:
       type: string
       description: User's fiscal code.
-      pattern: '^[A-Z]{6}\d{2}[A-Z]\d{2}[A-Z]\d{3}[A-Z]$'
+      pattern: '^[A-Z]{6}[0-9LMNPQRSTUV]{2}[ABCDEHLMPRST][0-9LMNPQRSTUV]{2}[A-Z][0-9LMNPQRSTUV]{3}[A-Z]$'
```

Comment:
question: are you sure this is the 'correct' regex for user Fiscal Codes? 

I mean, are we managing all the edge cases (i.e.: omocodia)?